lib/insights/api/common/application_controller_mixins/exception_handling.rb

```diff
@@ -35,6 +38,24 @@ def error_code_from_class(exception)
               DEFAULT_ERROR_CODE
             end
```

Comment:
I know this doesn't belong to your changes. Should we change it into string?

Reply:
@hsong-rh This still needs to be an integer because this is the HTTP status value which is always an integer, when we create our errors array we set the status and convert the into to string to comply with the JSON-API spec.

Reply:
Yes, I found the conversion `error_code_from_class(exc).to_s`